src/org/traccar/CopyAttributesHandler.java

```diff
@@ -29,8 +29,8 @@ private Position getLastPosition(long deviceId) {
 
     @Override
     protected Position handlePosition(Position position) {
-        String attributesString = Context.getDeviceManager().lookupConfigString(position.getDeviceId(),
-                "processing.copyAttributes", null);
+        String attributesString = Context.getDeviceManager().lookupAttributeString(position.getDeviceId(),
```

Comment:
I think it's more readable if it doesn't fit on one line, move all parameters to the next one.